types/lodash/lodash-tests.ts

```diff
@@ -3359,18 +3367,9 @@ namespace TestZipObject {
 }
 
 // _.zipWith
-namespace TestZipWith {
+{
     type TestZipWithFn = (a1: number, a2: number) => number;
 
-    {
-        let result: number[][];
-        result = _.zipWith([1, 2]);
```

Comment:
Some of these tests fail due to my changes. However, as stated earlier, there's no point in calling `zipWith` without an iteratee - in that case you should call `zip` instead.